assets/terraform-modules/bare-metal/flatcar-linux/kubernetes/cl/install.yaml.tmpl

```diff
@@ -40,9 +39,13 @@ storage:
             -C ${os_channel} \
             -V ${os_version} \
             -o "${container_linux_oem}" \
-            ${baseurl_flag} \
-            -i ignition.json
+            ${baseurl_flag}
           udevadm settle
+          OEM_DEV="$(blkid -t "LABEL=OEM" -o device)"
+          mkdir -p /tmp/oemfs
+          mount "$${OEM_DEV}" /tmp/oemfs
+          echo 'set linux_append="${kernel_args} ignition.config.url=${ignition_endpoint}?{{.request.raw_query}}&os=installed"' | tee -a /tmp/oemfs/grub.cfg
```

Comment:
What happens when ignition endpoint becomes unavailable? Flatcar will still be able to boot?

Reply:
Yes, Ignition will only query it if it runs and by default it only runs once. `touch /boot/flatcar/first_boot` will allow to trigger it again.